deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
Do you think it is confusing to assume everyone is deploying this as a cluster add-on?  If they don't take that additional step - their service will be shutdown.  On the other hand, if they **should** be deployed this way, should _grafana_, _chronograf_, and _influxdb_ also be flagged with `cluster-service` as well?